src/features/pass-survey/lib/context/index.ts

```diff
@@ -0,0 +1 @@
+export { default as ActivityItemIdentityContext } from './ActivityItemIdentityContext';
```

Comment:
Please rename the folder name: context => contexts

Reply:
done